.github/workflows/continuous-integration.yaml

```
@@ -0,0 +1,121 @@
+        uses: actions/cache@v4
+        with:
+          path: server/node_modules
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('server/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+
+      - name: Run E2E tests
+        working-directory: server
+        run: npm run test:e2e -- --ci
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)